controllers/user.js

```diff
@@ -568,6 +550,7 @@ async function getBreachStats(req, res) {
 
 
 function logout(req, res) {
+  // Growth Experiment
   if (AppConstants.EXPERIMENT_ACTIVE) {
```

Comment:
Unreachable Code – This is gated on the ENV variable, `EXPERIMENT_ACTIVE` set to 1.

Reply:
nit: this doesn't use a type-safe value check like above? `AppConstants.EXPERIMENT_ACTIVE === "1"` So `EXPERIMENT_ACTIVE=false` would evaluate to `true` here. (❤️ Javascript)

Reply:
This is actually a bug from the experiment. Adding back the logic for `EXPERIMENTS_ENABLED` to gate it correctly.